cmake/CMakeLists.txt

```diff
@@ -85,6 +85,9 @@ option(onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS "Dump node input shapes and output
 option(onnxruntime_USE_DML "Build with DirectML support" OFF)
 option(onnxruntime_USE_ACL "Build with ACL support" OFF)
 
+# winml options
+option(winml_use_telemetry "Build WinML with Telemetry" OFF)
```

Comment:
>winml_use_telemetry [](start = 7, length = 19)

since this is used for both ort and winml, let's call it onnxrutime_use_telemetry